### DISCUSSION TITLE
Overwrite global.css file

### DISCUSSION THREAD

**Discussion by kogeletey** [chakra-ui/panda#1210] [Ideas]
I would like to be able to rewrite the global css file to my own.

**Comment by anubra266** [comment]
I don't understand your question @kogeletey. What's preventing you from overwriting global cs?

**Reply by kogeletey** [reply]
The file generate a bunch of css variables that I don't need to my work, like a 
```css
   --blur:   ;
    --brightness:   ;
    --contrast:   ;
    --grayscale:   ;
    --hue-rotate:   ;
    --invert:   ;
    --saturate:   ;
    --sepia:   ;
    --drop-shadow:   ;
```

So I would like to remove them and put my own instead of this file.

**Reply by astahmer** [reply]
there's a hook (from [`config.hooks`](https://github.com/chakra-ui/panda/blob/4799d59f049d8bc513855f3988ae82addc477e7c/packages/types/src/hooks.ts#L32)) that you could use for that : `generator:css` when the first argument is `global.css`
